Parsers/ASimProcessEvent/ProductParsers/ProcessEventMicrosoftLinuxSysmonTerminate.yaml

```diff
@@ -0,0 +1,51 @@
+Parser:
+  Title: ASIM Sysmon/Linux Process Termination Event Parser
+  Version: '0.1'
+  LastUpdated: Sep 26, 2021
+Product:
+  Name: Microsoft Linux Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
+References:
+- Title: ASIM Process Schema
+  Link: https://aka.ms/AzSentinelProcessEventsDoc
+- Title: ASIM
+  Link: https:/aka.ms/AzSentinelNormalization
+Description: ASIM Sysmon/Linux Process Termination Event Parser (event number 5)
+ParserName: vimProcessTerminateLinuxSysmon
+ParserQuery: | 
+            let ParsedProcessEvent=(){
+            Syslog
+            | where SyslogMessage has_all ('<Provider Name="Linux-Sysmon"', '<EventID>5</EventID>')
+            | parse SyslogMessage with * '<Data Name="RuleName">'RuleName'</Data>'
+                                        '<Data Name="UtcTime">'UtcTime'</Data>'
+                                        '<Data Name="ProcessGuid">{'ProcessGuid'}</Data>'
+                                        '<Data Name="ProcessId">'ProcessId'</Data>'
+                                        '<Data Name="Image">'Image'</Data>' new_ver_data
+            | parse new_ver_data with '<Data Name="User">'ActorUsername '</Data>' *
```

Comment:
again: finishing the previous with "*" and starting this with "*" and not using new_ver_data is more efficient.